components/Buttons/examples/resources/ButtonsContentEdgeInsets.storyboard

```diff
@@ -21,23 +21,23 @@
                         <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
-                            <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="bvE-5P-0PG" customClass="MDCFlatButton">
-                                <rect key="frame" x="115.5" y="83" width="144" height="34"/>
+                            <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="bvE-5P-0PG" customClass="MDCButton">
+                                <rect key="frame" x="104.5" y="83" width="166" height="38"/>
```

Comment:
Why the repositioning?

Reply:
Interface builder nudged the view when it was changed to a system type.

Reply:
Why do they need to be system buttons now? Is this something we should document in the READMEs?

Reply:
jverkoey@'s previous comment was updated. Why do they need to be *custom* buttons now? Will our clients need to update all of their storyboards/xibs as well?

Reply:
They don't need to be system buttons, that was a typo in my original comment.

Reply:
The IB file was marking the buttons as system buttons, which they are not. They have to be marked as custom buttons.

Reply:
If they have to be marked as custom buttons, how were they previously broken?

Reply:
Buttons have always had to be initialized as custom buttons. If this is not clearly documented in our buttons readme then we will need to clearly note that.

Reply:
Sorry, I was unintentionally unclear. In what way do our buttons break if they are marked as "system" buttons in IB?  I thought IB will just be calling alloc/init and then setting KVC properties on the instance?

Reply:
The incorrect behavior was originally noted in https://github.com/material-components/material-components-ios/pull/3495. Whenever an MDCButton is initialized as a system button the title text color will get tinted incorrectly.

Reply:
That makes sense. Can you please mention that change as well in the commit message?

Reply:
Opened #3586 to track the Docs update.

Reply:
Done.